### DISCUSSION TITLE
Resolved: Whats the plan for `bitcoin-io`

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#3203] [General]
We have quite a few discussions running and I have no idea what the plan is, or what the next step is.

- https://github.com/rust-bitcoin/rust-bitcoin/pull/3193
- https://github.com/rust-bitcoin/rust-bitcoin/issues/3172
- https://github.com/rust-bitcoin/rust-bitcoin/pull/3181
- https://github.com/rust-bitcoin/rust-bitcoin/pull/3173
- https://github.com/rust-bitcoin/rust-bitcoin/pull/3191
- https://github.com/rust-bitcoin/rust-bitcoin/pull/3190

**Comment by apoelstra** [comment]
#3181 is definitely going in once I get a chance to review it (it's very long). then I think #3191 is a straightforward followup from there.

#3173 is probably going in but we still need to convince Kix that we should.

The other two PRs are drafts and I think just demos of ideas that aren't intended to be merged as-is.

**Reply by Kixunil** [reply]
#3193 is meant to be serious. I almost have it completed. I've changed a few things there and we will actually take advantage of buffering.

**Comment by tcharding** [comment]
#3181 is easy enough to review, it should be all the same changes we reviewed already in #3176 except for the last patch, I did not verify that though I just quickly looked and acked.

**Comment by tcharding** [comment]
Marked resolved and closing because this doesn't add much value any more because its so old.
